Source/WebKit/Configurations/WebContentService.xcconfig

```diff
@@ -30,6 +30,13 @@ PRODUCT_NAME_Development = com.apple.WebKit.WebContent.Development;
 PRODUCT_NAME_CaptivePortal = com.apple.WebKit.WebContent.CaptivePortal;
 
 PRODUCT_BUNDLE_IDENTIFIER = $(PRODUCT_NAME);
+PRODUCT_BUNDLE_IDENTIFIER[sdk=iphonesimulator*] = $(PRODUCT_NAME).Service;
+PRODUCT_BUNDLE_IDENTIFIER[sdk=iphonesimulator17.0*] = $(PRODUCT_NAME);
+PRODUCT_BUNDLE_IDENTIFIER[sdk=iphonesimulator17.1*] = $(PRODUCT_NAME);
+PRODUCT_BUNDLE_IDENTIFIER[sdk=iphonesimulator17.2*] = $(PRODUCT_NAME);
+PRODUCT_BUNDLE_IDENTIFIER[sdk=iphonesimulator17.3*] = $(PRODUCT_NAME);
+PRODUCT_BUNDLE_IDENTIFIER[sdk=iphonesimulator18.0*] = $(PRODUCT_NAME);
```

Comment:
This pattern doesn't seem very maintainable. We should think about how to make this more automatic.

Reply:
That is a good point! We may be able to remove most of this as we move forward.

Thanks for reviewing!